plugins/woocommerce/client/admin/client/blueprint/components/get-option-groups.ts

```diff
@@ -0,0 +1,162 @@
+	woocommerce_erasure_request_removes_order_data: 'Accounts',
+	woocommerce_erasure_request_removes_download_data: 'Accounts',
+	woocommerce_allow_bulk_remove_personal_data: 'Accounts',
+	woocommerce_registration_privacy_policy_text: 'Accounts',
+	woocommerce_checkout_privacy_policy_text: 'Accounts',
+	woocommerce_delete_inactive_accounts: 'Accounts',
+	woocommerce_trash_pending_orders: 'Accounts',
+	woocommerce_trash_failed_orders: 'Accounts',
+	woocommerce_trash_cancelled_orders: 'Accounts',
+	woocommerce_anonymize_refunded_orders: 'Accounts',
+	woocommerce_anonymize_completed_orders: 'Accounts',
+	woocommerce_email_from_name: 'Emails',
+	woocommerce_email_from_address: 'Emails',
+	woocommerce_email_header_image: 'Emails',
+	woocommerce_email_base_color: 'Emails',
+	woocommerce_email_background_color: 'Emails',
+	woocommerce_email_body_background_color: 'Emails',
+	woocommerce_email_text_color: 'Emails',
+	woocommerce_email_footer_text: 'Emails',
+	woocommerce_email_footer_text_color: 'Emails',
+	woocommerce_email_auto_sync_with_theme: 'Emails',
+	woocommerce_merchant_email_notifications: 'Emails',
+	woocommerce_coming_soon: 'Site visibility',
+	woocommerce_store_pages_only: 'Site visibility',
+	woocommerce_cart_page_id: 'Advanced',
+	woocommerce_checkout_page_id: 'Advanced',
+	woocommerce_myaccount_page_id: 'Advanced',
+	woocommerce_terms_page_id: 'Advanced',
+	woocommerce_checkout_pay_endpoint: 'Advanced',
+	woocommerce_checkout_order_received_endpoint: 'Advanced',
+	woocommerce_myaccount_add_payment_method_endpoint: 'Advanced',
+	woocommerce_myaccount_delete_payment_method_endpoint: 'Advanced',
+	woocommerce_myaccount_set_default_payment_method_endpoint: 'Advanced',
+	woocommerce_myaccount_orders_endpoint: 'Advanced',
+	woocommerce_myaccount_view_order_endpoint: 'Advanced',
+	woocommerce_myaccount_downloads_endpoint: 'Advanced',
+	woocommerce_myaccount_edit_account_endpoint: 'Advanced',
+	woocommerce_myaccount_edit_address_endpoint: 'Advanced',
+	woocommerce_myaccount_payment_methods_endpoint: 'Advanced',
+	woocommerce_myaccount_lost_password_endpoint: 'Advanced',
+	woocommerce_logout_endpoint: 'Advanced',
+	woocommerce_api_enabled: 'Advanced',
+	woocommerce_allow_tracking: 'Advanced',
+	woocommerce_show_marketplace_suggestions: 'Advanced',
+	woocommerce_custom_orders_table_enabled: 'Advanced',
+	woocommerce_custom_orders_table_data_sync_enabled: 'Advanced',
+	woocommerce_analytics_enabled: 'Advanced',
+	woocommerce_feature_rate_limit_checkout_enabled: 'Advanced',
+	woocommerce_feature_order_attribution_enabled: 'Advanced',
+	woocommerce_feature_site_visibility_badge_enabled: 'Advanced',
+	woocommerce_feature_remote_logging_enabled: 'Advanced',
+	woocommerce_feature_email_improvements_enabled: 'Advanced',
+	woocommerce_feature_blueprint_enabled: 'Advanced',
+	woocommerce_feature_product_block_editor_enabled: 'Advanced',
+	woocommerce_hpos_fts_index_enabled: 'Advanced',
+	woocommerce_feature_cost_of_goods_sold_enabled: 'Advanced',
+};
```

Comment:
I can't remember if I mentioned this to you already @moon0326, but the method by which the new Settings gathers settings configurations could be useful here. The configuration objects contain an `id` that corresponds to the option name and it would all be grouped under the appropriate option group. You could use that object to gather all Settings ids, even those added by extensions, and have a more complete list.

Reply:
https://github.com/woocommerce/woocommerce/blob/38fb1f9839b865e9f488c72d9fbba3eb3745dce9/plugins/woocommerce/src/Admin/Features/Settings/Init.php#L178